src/managers/input/InputManager.cpp

```diff
@@ -1177,11 +1160,16 @@ void CInputManager::setTabletConfigs() {
     for (auto& t : m_lTablets) {
         const auto HASCONFIG = g_pConfigManager->deviceConfigExists(t.name);
 
-        if (HASCONFIG) {
-            const auto OUTPUT   = g_pConfigManager->getDeviceString(t.name, "output");
-            const auto PMONITOR = g_pCompositor->getMonitorFromString(OUTPUT);
+        if (wlr_input_device_is_libinput(t.wlrDevice)) {
+            const auto LIBINPUTDEV = (libinput_device*)wlr_libinput_get_device_handle(t.wlrDevice);
+
+            const int  ROTATION =
+                std::clamp(HASCONFIG ? g_pConfigManager->getDeviceInt(t.name, "transform") : g_pConfigManager->getInt("input:tablet:transform"), 0, 7);
```

Comment:
oopsie - you added input:tablet:transform without actually making the config option - this will crash!